modules/ROOT/pages/exportpdf.adoc

```diff
@@ -0,0 +1,59 @@
+= Export to PDF plugin
+:navtitle: Export to PDF
+:description: The Export to PDF feature provides the ability to generate a PDF file directly from the editor.
+:description_short: Generate a PDF file directly from the editor.
+:keywords: plugin, exportpdf, export to pdf
+:pluginname: Export to PDF
+:plugincode: exportpdf
+:plugincategory: premium
+
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The {pluginname} functionality gathers the HTML produced using the `editor.getcontent()` method and the default editor content styles, along with the styles specified in the configuration options for the plugin. This data is transmitted to the {productname} Cloud Services HTML to PDF converter service. The service processes this information to create a PDF file, which is then sent back to the user's browser for saving to their local disk.
+
+== Interactive example
+
+[NOTE]
+.{pluginname} demo
+====
+Demos containing the {pluginname} plugin are currently only available on our main website. Check out https://www.tiny.cloud/tinymce/features/export-pdf/[the {pluginname} demo], or the https://www.tiny.cloud/[demo on our home page], to try it out today.
+====
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportpdf',
+  toolbar: 'exportpdf',
+  exportpdf_service_url: '<service url>'
+});
+----
+
+== Options
+
+The following configuration options affect the behavior of the {pluginname} plugin.
+
+include::partial$configuration/{plugincode}.adoc[][leveloffset=+1]
```

Comment:
```suggestion
include::partial$configuration/{plugincode}.adoc[leveloffset=+1]
```
fix heading levels in the navbar
before:
<img width="304" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/40410898/8964fd15-47e6-405a-a603-53e57f53298b">

after:
<img width="271" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/40410898/1710e48e-648b-4933-ae2f-19f62d0bb491">